core/pkg/server/sender.go

```diff
@@ -991,7 +995,11 @@ func (s *Sender) sendFiles(_ *service.Record, filesRecord *service.FilesRecord)
 		if strings.HasPrefix(file.GetPath(), "media") {
 			file.Type = service.FilesItem_MEDIA
 		}
-		s.sendFile(file)
+		s.wg.Add(1)
+		go func(file *service.FilesItem) {
+			s.sendFile(file)
+			s.wg.Done()
+		}(file)
```

Comment:
noice!

Reply:
now we are a little better than python need to keep going :)